docs/launcher.md

```diff
@@ -0,0 +1,86 @@
+
+# vLLM Launcher
+
+## Description
+
+This document describes a vLLM launcher to be used in model swapping techniques. The launcher initializes and manages various subsidiary inference vLLM servers and keep an inventory with it's ID's and the model (in addition to other information like the type of accelerator and the GPU number). The launcher does basic vLLM code loading and initialization work (like import modules) of the inference server so that this work does not have to be done at the startup of the inference server process, reducing the startup latency. In future work, the launcher could be extended to sleep and wake up the inference servers that have been initialize by its POST method.
```

Comment:
> GPU number

Does this launcher only work with single-GPU servers? I had expected it could support using more than one GPU of the node. I had expected it could work with other sorts of accelerators.